src/main/java/de/retest/web/FrameConverter.java

```diff
@@ -36,7 +36,7 @@ public FrameConverter( final String queryJs, final RetestIdProvider retestIdProv
 	public void addChildrenFromFrames( final WebDriver driver, final RootElement lastChecked ) {
 		final List<Element> frames =
 				de.retest.web.selenium.By.findElements( lastChecked.getContainedElements(), isFrame() );
-		log.debug( "Found {} frame(s), getting data per frame.", frames.size() );
+		log.debug( "Found {} sub-frame(s), getting data per frame.", frames.size() );
```

Comment:
Omit the confusing log entirely if there are no frames?

```java
if ( frames.size() > 0 ) {
    log.debug( "Found {} frame(s), getting data per frame.", frames.size() );
}
```

Reply:
Hmm. Dann aber if für die ganze Methode, sonst wird das unverständlich.